configs/config-combinations/readme.md

```diff
@@ -0,0 +1,19 @@
+This directory containers YAML files which are config combinations for various
```

Comment:
I'm not sure I like the "config combination" name, some alternatives:
config declaration
config tracking declaration
config generation 
container config declaration
container config spec
config spec

Reply:
Changed to dependency tracking for configs